cms/static/js/views/baseview.js

```diff
@@ -26,6 +26,14 @@ define(["jquery", "underscore", "backbone", "gettext", "js/utils/handle_iframe_b
             //override the constructor function
             constructor: function(options) {
                 _.bindAll(this, 'beforeRender', 'render', 'afterRender');
+
+                // Merge passed options and view's options property and
+                // attach to the view's options property
+                if (this.options) {
+                    options = _.extend({}, _.result(this, 'options'), options);
+                }
```

Comment:
@ehteshamkafeel Any special reason for doing all the above. What about just 
`this.options = options;`
OR
`this.options = _.extend({}, options)`